src/unwinder/arm64/unwinder_arm64.cpp

```diff
@@ -215,7 +215,11 @@ Return Value:
         SourceAddress = StartingSp + FIELD_OFFSET(ARM64_KTRAP_FRAME, X);
         for (RegIndex = 0; RegIndex < 18; RegIndex++) {
             UPDATE_CONTEXT_POINTERS(UnwindParams, RegIndex, SourceAddress);
+#ifdef __clang__
```

Comment:
Think this should maybe be `FEATURE_PAL` not `__clang__`? Besides that on ARM the Clang version is used universally, not sure if that'd be preferred here too.